.env.template

```diff
@@ -1,6 +1,7 @@
 THE_MOVIE_DB_API = "<<Your MovieDB API Token>>"
 # A trailing slash is required
-DEBUG_HOST = "<<Your Debbuging Server Host>>"
-RELEASE_HOST = "<<Your Production Server Host>>"
+DEV_HOST = "<<Your Debbuging Server Host>>"
+ALPHA_HOST = "<<Your Alpha Server Host>>"
```

Comment:
Right now echo `RELEASE_HOST=$DEBUG_HOST` is used in workflows when building an alpha. 
Plus I think that workflows are not updated to the new `.env` file

Reply:
Workflows are updated in this PR. We will juste need to rename secrets.